frontend/.eslintrc.json

```diff
@@ -1,40 +0,0 @@
-{
-  "env": {
-    "browser": true,
-    "es2021": true
-  },
-  "ignorePatterns": [
-    "*.js",
-    "*.config.js",
-    "dist/**/*",
-    "**/*.css",
-    "**/*.scss",
-    "**/*.md"
-  ],
-  "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react/recommended",
-    "plugin:prettier/recommended",
-    "plugin:storybook/recommended"
```

Comment:
included in new config - definitely need to steal this one for open-forms-sdk too!